### PR TITLE
Bug/bitpay card

### PR DESCRIPTION
**Pull request by matiu**
Refactor bitpay-card storage.
history/balance cache.

fixes: https://github.com/bitpay/copay/issues/5489

### CODE REVIEW

**Comment by JDonadio** [comment]
Refresh balance issue related [here](https://github.com/bitpay/copay/issues/5489)

**Comment by matiu** [comment]
ready to be reviewed.

File: src/js/services/amazonService.js [hunk 1]
```diff
@@ -1,24 +1,36 @@
...
-      credentials.BITPAY_API_URL = "https://test.bitpay.com";
-    } else {
-      credentials.BITPAY_API_URL = "https://bitpay.com";
-    };
+  /*
+   * Development: 'testnet'
+   * Production: 'livenet'
+   */
+  //credentials.NETWORK = 'livenet';
+  credentials.NETWORK = 'testnet';
```
Comment (cmgustavo) [inline src/js/services/amazonService.js]: Should be `livenet`

File: src/js/services/glideraService.js [hunk 2]
```diff
@@ -16,7 +17,8 @@ angular.module('copayApp.services').factory('glideraService', function($http, $l
      * Development: 'testnet'
      * Production: 'livenet'
      */
-    credentials.NETWORK = 'livenet';
+    //    credentials.NETWORK = 'livenet';
+    credentials.NETWORK = 'testnet';
```
Comment (cmgustavo) [inline src/js/services/glideraService.js]: Should be `livenet`

**Pull request merged**